Automattic/wp-calypso
# Datacenter Picker: Indicate primary data center in the 'Web Server Settings' UI element

Fixes https://github.com/Automattic/dotcom-forge/issues/1238

## Proposed Changes

Building on https://github.com/Automattic/wp-calypso/pull/70237, we now show the primary data center in the 'Web Server Settings' UI element:

<img width="767" alt="image" src="https://user-images.githubusercontent.com/36432/204576976-84ee0b86-fe94-4ec4-9c05-c3b0307c03f8.png">

<img width="786" alt="image" src="https://user-images.githubusercontent.com/36432/204578452-8f156be3-e970-4095-ab98-920ab31a4add.png">

## Testing Instructions

1. Apply D93602-code
2. Navigate to 'Hosting Configuration' on an Atomic site.
3. Verify the expected primary data center is selected.
